### PR TITLE
Capitalized beginning of line messages, added periods

### PR DESCRIPTION
**Pull request by RichardLitt**
Created due to: https://github.com/ipfs/go-ipfs/pull/2228#discussion_r50497930

License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: core/commands/object/object.go [hunk 1]
```diff
@@ -326,10 +326,10 @@ and then run
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.FileArg("data", true, false, "Data to be stored as a DAG object").EnableStdin(),
+		cmds.FileArg("data", true, false, "Data to be stored as a DAG object.").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("inputenc", "Encoding type of input data, either \"protobuf\" or \"json\""),
+		cmds.StringOption("inputenc", "Encoding type of input data, either \"protobuf\" or \"Json\"."),
```
Comment (whyrusleeping) [inline core/commands/object/object.go]: this one should stay lowercase. Its the actual value we're expecting to be typed in

**Comment by whyrusleeping** [comment]
One comment, then LGTM

**Comment by RichardLitt** [comment]
Fixed.

**Comment by rht** [comment]
+1 LGTM

**Pull request merged**